.github/workflows/fossa.yaml

```diff
@@ -102,19 +102,24 @@ jobs:
           echo '```' >> $GITHUB_STEP_SUMMARY
 
       - name: List targets
+        id: list-targets
         continue-on-error: true
         env:
           FOSSA_API_KEY: ${{ secrets.fossa-api-key }}
         run: |
           echo '### Targets' >> $GITHUB_STEP_SUMMARY
           echo '' >> $GITHUB_STEP_SUMMARY
           echo '```bash' >> $GITHUB_STEP_SUMMARY
-          fossa list-targets 2>&1 | tee >(grep -v '\[ ' >> $GITHUB_STEP_SUMMARY)
+          TARGETS=$(fossa list-targets 2>&1)
+          echo $TARGETS | tee >(grep -v '\[ ' >> $GITHUB_STEP_SUMMARY)
           echo '```' >> $GITHUB_STEP_SUMMARY
-
+          TARGETS_COUNT=$(echo $TARGETS | grep -v '\[ ' | wc -l)
+          echo "$TARGETS_COUNT" >> "$GITHUB_OUTPUT"
```

Comment:
Will those conditionals work, or should it be like the following?
```
echo "targets_count=${TARGETS_COUNT}" >> $GITHUB_OUTPUT
```

Reply:
<img src="https://media3.giphy.com/media/AjYsTtVxEEBPO/giphy.gif"/>